### DISCUSSION TITLE
Buyer Info and Shipping Address

### DISCUSSION THREAD

**Discussion by ayotade** [abuzuhri/Amazon-SP-API-CSharp#719] [Q&A]
Hi,

My application pulled through 100 out of 130 orders. The remaining orders are not returning buyer info and some shipping address fields.

Any idea what the issue is please?

**Comment by ayotade** [comment]
They have now.

thanks

**Comment by marcsavall** [comment]
can you upload code examples to recover PII personal data, using Amazon-SP-API-CSharp : address, phone, email...
TY.

**Comment by optitsolutions** [comment]
Sehr geehrte Damen und Herren,

bis inklusive 03.04.2024 bin ich nicht im B++ro und lese meine Mails nur unregelm+ñ+ig.

Bitte wenden Sie sich in dringenden F+ñllen an das B++ro unter ***@***.***

Ihre Mail wird nicht automatisch weitergeleitet.




Mit freundlichen Gr+++en

Thorsten Altmann






can you upload code examples to recover PII personal data, using Amazon-SP-API-CSharp : address, phone, email...
TY.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/abuzuhri/Amazon-SP-API-CSharp/discussions/719#discussioncomment-8929515
